jobs/jobs.go

```diff
@@ -12,6 +15,7 @@ const (
 	ResponderCommentsSyncSchedule   = "@every 1h"
 	ResponderConfigSyncSchedule     = "@every 1h"
 	CleanupJobHistoryTableSchedule  = "@every 24h"
+	upstreamPushSchedule            = "@every 5s"
```

Comment:
@moshloop This is 5s only for testing purpose. But what should be the final value? Maybe this could also be a flag ... ?

Reply:
This shouldn't be a schedule, but a loop - the changelog table acts as an event queue: https://github.com/flanksource/incident-commander/blob/main/events/events.go#L67